src/DssSpell.t.base.sol

```diff
@@ -1689,14 +1508,12 @@ contract DssSpellTestBase is Config, Test {
         }
     }
 
-    // Add an exception here if a registered deployer can be a valid auth on target
-    function skipWards(address target, address deployer) internal pure returns (bool) {
-        // Add logic here in case any wards need to be skipped, otherwise return false.
+    function _skipWards(address target, address deployer) internal pure returns (bool) {
```

Comment:
why don't we just remove this? If we are keeping it, maybe the comments are good enough to state or even could be improved to explain why we are keeping this function in mainnet?

Reply:
Comment added. If we want to remove it we might want to add a separate PR. I think I slightly prefer this way.

Reply:
which way? having the comment? (i like your comment and am good with just keeping that, just not sure which way you're thinking)

Reply:
sounds good to me with your added comment